lib/bandit/http1/adapter.ex

```diff
@@ -367,12 +373,28 @@ defmodule Bandit.HTTP1.Adapter do
     415 => "Unsupported Media Type",
     416 => "Requested range not satisfiable",
     417 => "Expectation Failed",
+    418 => "I'm a teapot",
+    421 => "Misdirected Request",
+    422 => "Unprocessable Entity",
+    423 => "Locked",
+    424 => "Failed Dependency",
+    425 => "Too Early",
+    426 => "Upgrade Required",
+    428 => "Precondition Required",
+    429 => "Too Many Requests",
+    431 => "Request Header Fields Too Large",
+    451 => "Unavailable For Legal Reasons",
     500 => "Internal Server Error",
     501 => "Not Implemented",
     502 => "Bad Gateway",
     503 => "Service Unavailable",
     504 => "Gateway Time-out",
-    505 => "HTTP Version not supported"
+    505 => "HTTP Version Not Supported",
+    506 => "Variant Also Negotiates",
+    507 => "Insufficient Storage",
+    508 => "Loop Detected",
+    510 => "Not Extended",
+    511 => "Network Authentication Required"
```

Comment:
Not in 9110

Reply:
[RFC 6585](https://www.rfc-editor.org/rfc/rfc6585#section-6)